tests/peft_integration/test_peft_integration.py

```diff
@@ -845,7 +845,7 @@ def test_peft_pipeline_no_warning(self):
         # Create pipeline with PEFT model while capturing log output
         # Check that the warning message is not present in the logs
         pipeline_logger = logging.get_logger("transformers.pipelines.base")
-        with self.assertNoLogs(pipeline_logger, logging.ERROR) as cl:
+        with self.assertNoLogs(pipeline_logger, logging.ERROR):
```

Comment:
I will leave you to judge which version to use. If you are confident to use `with self.assertNoLogs`, OK for me too.

Reply:
I think asserting that there are no logs is enough, when there are no logs, there is no further need to check that the error message does not appear in the logs.